pinot-controller/src/main/java/org/apache/pinot/controller/validation/OfflineSegmentIntervalChecker.java

```diff
@@ -136,6 +136,20 @@ private void validateOfflineSegmentPush(TableConfig tableConfig) {
     _validationMetrics.updateSegmentCountGauge(offlineTableName, numSegments);
   }
 
+  @Override
+  protected void nonLeaderCleanup(List<String> tableNamesWithType) {
+    for (String tableNameWithType : tableNamesWithType) {
+      TableType tableType = TableNameBuilder.getTableTypeFromTableName(tableNameWithType);
+      if (tableType == TableType.OFFLINE) {
+        _validationMetrics.cleanupMissingSegmentCountGauge(tableNameWithType);
```

Comment:
Can you add at least `TODO` comment to clean this up? We can simply loop through all the enum values from `ValidationMetricName` and clean up the metrics.